powersimdata/data_access/data_access.py

```diff
@@ -371,7 +408,7 @@ def move_to(self, file_name, to_dir=None, change_name_to=None):
 
         os.remove(from_path)
 
-    def execute_command(self, command):
+    def _execute_command(self, command):
         """Execute a command locally at the data access.
```

Comment:
Execute a command on the server?

Also, do we need to have a function? I mean it used throughout the class but what is the gain of doing `self._execute_command(command)` and not  `self.ssh.exec_command(command)`? The method could then be removed from the parent class.